vectorizers/_vectorizers.py

```diff
@@ -269,15 +271,7 @@ def preprocess_token_sequences(
         )
         for sequence in token_sequences
     ]
-    """
-    result_sequences = []
-    for sequence in token_sequences:
-        result_sequence = np.array(
-            [token_dictionary[token] for token in sequence if token in token_dictionary]
-        )
-        if len(result_sequence) > 0:
-            result_sequences.append(result_sequence)
-    """
+
```

Comment:
We aren't converting to integer sequences anymore? Did I miss something?

Reply:
This was already commented out and I just deleted it.  I think the pre-process_token_sequences does the conversion and so we don't need it here .

Reply:
Ah, fair. Good call then.